.github/workflows/consecutive.yml

```diff
@@ -34,7 +34,7 @@ on:
 
 env:
   JOBS_NUMBER: "8"
-  NETWORK: teststand2
+  NETWORK: night-stand
```

Comment:
In the future, we need to rename this variable to something like DEFAULT_NETWORK or DEFAULT_ENV, as it's only used for getting the initial value and for comparison to a dynamic value